src/xFactor/Participant.java

```diff
@@ -0,0 +1,39 @@
+package xFactor;
+
+import java.util.ArrayList;
+
+public class Participant extends Person {
+	private ArrayList<String> qualities = new ArrayList<String>();
+	
+	private Competition competition = new Competition();
+	
+	public Participant() {}
+	
+	public Participant(String fName, String lName, int age, Gender gender, String birthPlace, int id) {
+		super(fName, lName, age, gender, birthPlace, id);
+		initializeParticipant();
+	}
+	
+	private void initializeParticipant(){
+		qualities.add("Singing");
+		competition.getParticipants().add(this);
+		
+	}
+	
+	public ArrayList<String> getQualities() {
+		return qualities;
+	}
+	
+	public void setQualities(ArrayList<String> qualities) {
+		this.qualities = qualities;
+	}
+
+	public void addQuality(String quality) {
+		this.qualities.add(quality);

```

Comment:
validate if qualities is null and initialize it here, do not initialize it on line 6